python/dicebeard/__init__.py

```diff
@@ -6,13 +6,15 @@
 from telepot import glance
 from telepot.namedtuple import InlineKeyboardMarkup, InlineKeyboardButton
 
-from skybeard.beards import BeardChatHandler, ThatsNotMineException
+from skybeard.beards import BeardChatHandler, ThatsNotMineException, BeardDBTable
 from skybeard.decorators import onerror, getargsorask, getargs
 
 from .skb_roll import roll
 from .helper import TrainResult, AnswerTimer
 from .utils import image_to_bytesio
 
+import matplotlib.pyplot as plot
```

Comment:
This is usually imported as `plt` (by convention). We should probably change it in future versions.